flexmeasures/api/v3_0/tests/test_assets_api.py

```diff
@@ -366,3 +365,113 @@ def test_get_assets_new(
     print("Server responded with:\n%s" % get_assets_response.json)
     assert get_assets_response.status_code == 200
     assert len(get_assets_response.json) == 1
+    assert get_assets_response.json[0]["name"] == "Test ConsultantClient Asset"
+
+
+@pytest.mark.parametrize(
+    "requesting_user", ["test_consultant_user@seita.nl"], indirect=True
+)
+def test_consultant_can_not_patch(
+    client,
+    setup_api_test_data,
+    setup_accounts,
+    requesting_user,
+):
+    """
+    Try to edit an asset belonging to the ConsultantClient account with the Consultant account.
+    The Consultant account only has read access.
+    """
+    consultant_client_asset = GenericAsset.query.filter_by(
+        name="Test ConsultantClient Asset"
+    ).one_or_none()
+    print(consultant_client_asset)
+
+    asset_edit_response = client.patch(
+        url_for("AssetAPI:patch", id=consultant_client_asset.id),
+        json={
+            "latitude": consultant_client_asset.latitude,
```

Comment:
More simply, set to a bogus value like 0. Reads easier (in the sense that we actually attempt to change the value)

Reply:
Changed the value to 0 so it's more clear the value is not important here.